src/components/ShopListContent/PriceFilter.js

```diff
@@ -37,7 +50,7 @@ function PriceFilter() {
           <Slider
             sx={CustomSliderStyles}
             className={classes.filterSlider}
-            value={value}
+            value={filterPrice || value}
```

Comment:
Try not to use multiple constant values for component prob using condition like this, instead split this to a variable. Eg: 

```javascript
// Using ?? instead of || because ?? meaning if filterPrice is null/undefine then using value
const slideFilterPrice = filterPrice ?? value;
<Slider ... value={slideFilterPrice} />
```